CodeSmartBiks/AD320-Group2-TeamProject
# Hdc 91 put api new order

With the recent master update from Crystals MenuSelection setup, I was able to fully implement my placing a new order PUT api. To fulling create a new order, both the orders table and the ordersdetails table need to be updated.  this required multiple queries, and a few levels of nested queries that caused issues. After Bob's suggestion, I spent several hours reading up and trail-and-error practicing with Stored Procedures, and eventually got it working in Postman.

To verify this:
 - confirm current orders and order details in the DB. (also check the /vendor page to see inprogress orders)
 - be sure to install the updated DB script, as it has the stored procedure that is called.  
 - go to /menu, and cart3's currently available menu items will be shown.
 - add 1 to 3 of ONE ITEM type, then hit Submit. _(we are not yet able to mix item types currently)_
 - check your orders and ordersdetails tables in the db again (also check the /vendor page to see the new order appear on the incomplete orders page).
